x/fees/genesis.go

```diff
@@ -0,0 +1,43 @@
+package fees
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authkeeper "github.com/cosmos/cosmos-sdk/x/auth/keeper"
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/tharsis/evmos/v3/x/fees/keeper"
+	"github.com/tharsis/evmos/v3/x/fees/types"
+)
+
+// InitGenesis import module genesis
+func InitGenesis(
+	ctx sdk.Context,
+	k keeper.Keeper,
+	accountKeeper authkeeper.AccountKeeper,
+	data types.GenesisState,
+) {
+	k.SetParams(ctx, data.Params)
+
+	// Ensure fees module account is set on genesis
+	if acc := accountKeeper.GetModuleAccount(ctx, types.ModuleName); acc == nil {
+		panic("the fees module account has not been set")
+	}
+
+	for _, fee := range data.DevFeeInfos {
+		contract := common.HexToAddress(fee.ContractAddress)
+		deployer, _ := sdk.AccAddressFromBech32(fee.DeployerAddress)
+		withdrawal, _ := sdk.AccAddressFromBech32(fee.WithdrawAddress)
```

Comment:
we should check if the `fee.WithdrawAddress` is empty or equal to the deployer's

Reply:
Right now, this is done in `keeper.SetFee`.